module/project/templates/helper.js

```diff
@@ -217,7 +217,7 @@ module.exports = class ProjectTemplatesHelper {
                 //     });
                 // }
 
-                parsedData.recommendedFor = recommendedFor;
+                // parsedData.recommendedFor = recommendedFor;
```

Comment:
@Prajwal17Tunerlabs test it

Reply:
i tested it now, it is working as expected